src/views/product-integration/component-view/index.tsx

```diff
@@ -9,6 +9,7 @@ import type { OutlineNavProps } from 'components/outline-nav'
 import { OutlineNavWithActive } from 'components/outline-nav/components'
 import VersionAlertBanner from 'components/version-alert-banner'
 import ProductIntegrationLayout from 'layouts/product-integration-layout'
+import useUserContentAnchorLinks from 'lib/hooks/use-user-content-anchor-links'
```

Comment:
Have added a call to `useUserContentAnchorLinks` on the `component-view` and `readme-view`. Figured this makes sense for now since our use cases for user content are relatively isolated. In the future, if we see this popping up on a lot of views, it might make sense to elevate it to say our core layout or something.

Reply:
I think this makes sense here right now; I agree with your decision!